cmd/snap-bootstrap/partition/encrypt.go

```diff
@@ -104,9 +132,9 @@ func cryptsetupFormat(key EncryptionKey, label, node string) error {
 		"luksFormat",
 		// use LUKS2
 		"--type", "luks2",
-		// key file read from stdin
+		// read key from stdin
 		"--key-file", "-",
-		// user Argon2 for PBKDF
+		// use Argon2i for PBKDF
```

Comment:
Without extra checks this doesn't tell me much. It would be good to explain what Argon2i is and why we picked it over alternatives.

Reply:
it's not really that relevant what we picked,  we are configuring it to do  virtually nothing (--iter-time) because key is not a passphrase but is already a random key with good entropy, a comment about that would be useful though

Reply:
argon2i is actually the default ATM (at least a compiled for ubuntu)